missions/W4/M1/Dockerfile

```
@@ -0,0 +1,89 @@
+    echo "export SPARK_VERSION=${SPARK_VERSION}" | sudo tee -a /etc/profile.d/hadoop.sh && \
+    echo "export HADOOP_HOME=${HADOOP_HOME}" | sudo tee -a /etc/profile.d/hadoop.sh && \
+    echo "export SPARK_HOME=${SPARK_HOME}" | sudo tee -a /etc/profile.d/hadoop.sh && \
+    echo "export PATH=\${PATH}:\${HADOOP_HOME}/bin:\${SPARK_HOME}/bin" | sudo tee -a /etc/profile.d/hadoop.sh && \
+    sudo chmod +x /etc/profile.d/hadoop.sh
+
+# hduser의 .bashrc에도 추가
+# RUN echo "source /etc/profile.d/hadoop.sh" >> /home/hduser/.bashrc && \
+#     chown hduser:hduser /home/hduser/.bashrc
+
+# pip, PySpark 설치 TODO: 나중에 합치자.
+RUN sudo apt-get install -y software-properties-common \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)